semgrep_output_v1.atd

```diff
@@ -2033,6 +2033,7 @@ type ci_scan_results = {
 
    searched_paths: fpath list;
    renamed_paths: fpath list;
+   ?skipped_paths <doc text="Files detected but not scanned due to errors (timeout, OOM, etc.). The app should NOT mark findings in these files as fixed.">: fpath list option;
```

Comment:
No objection to this being an option, but, `searched_paths`, `renamed_paths`, etc are just lists, which means the caller doesn't need to check for `None` before unpacking.  Seems like `None` and `Some []` have the same meaning, so we could follow suit with the other path lists?